### ISSUE TITLE
Is hasManyResource searchable?

### ISSUE THREAD

**Issue by rzenkov** [tailflow/laravel-orion#98]
Hi there

I can't find any docs or issues about this behavior. 

### Issue:

i've registered  `Orion::hasManyResource()`, 
for `class CategoryProductsController extends RelationController`

`/search` - endpoint exposed in `artisan route:list`

but, when i try to send request, i've got an error

```
  "message": "Method App\\Http\\Controllers\\Api\\CategoryProductsController::search does not exist.",
  "exception": "BadMethodCallException",
```

### Questions:

- Is this a bug?
- Or `search` method  should be created  by me?
- Can I disable `search` endpoint?

**Comment by rzenkov** [comment]
I've created `search` method in controller:
```    
    public function search(\Orion\Http\Requests\Request $request, $category)
    {
        return $this->index($request, $category);
    }
```

And it works fine.  I think It should be documented. ( but not  with my english ).

**Comment by alexzarbn** [comment]
Hi @rzenkov,

Thank you for raising this issue! It is actually a bug, and it is now fixed in [v2.0.1](https://github.com/tailflow/laravel-orion/releases/tag/2.0.1).
